modules/special-rules/nav.adoc

```diff
@@ -76,9 +93,9 @@
   *** xref:rearm-and-refuel.adoc[]
   *** xref:ground-attack.adoc[]
   *** xref:transport.adoc[]
-  *** xref:evac-evac.adoc[]
-  *** xref:counter-strike.adoc[]
+  *** xref:evac-evac.adoc[] OPTIONAL
+  *** xref:counter-strike.adoc[] OPTIONAL
```

Comment:
You've annotated these with OPTIONAL, but what are your thoughts about what we should do with these topics/rules?

Reply:
Move them to the Experimental module. I may have misunderstood something, but I thought that was where all the experimental and optional rules were being moved?

Reply:
I think I'd forgotten that those rules were published as optional, and considered them as effectively 'official' — which they could be for Remastered. But I'm happy enough to stick them in Experimental section for now and reflect on them later.